XamlStyler.Package/StylerPackage.cs

```diff
@@ -186,6 +183,9 @@ private void Execute(Document document)
                 stylerOptions.ConfigPath = configPath;
             }
 
+            stylerOptions.IndentSize = Int32.Parse(xamlEditorProps.Item("IndentSize").Value.ToString());
```

Comment:
Why Parse().ToString? Looks strange. Also TryParse and use a sensible default on failure feels like a decent alternative here.